src/languageservice/services/yamlCompletion.ts

```diff
@@ -1612,7 +1623,8 @@ export class YamlCompletion {
     separatorAfter: string,
     collector: CompletionsCollector,
     settings: StringifySettings,
-    arrayDepth = 0
+    arrayDepth = 0,
+    isArray?: boolean
```

Comment:
there are few others calls `collectDefaultSnippets()` without isArray parameter. If it's intended that these calls don't send `isArray` parameters it should be better to sed default value for that param

```suggestion
    isArray = false
```